docs/use_cases/model_applications/s2s_mid_lat/UserScript_obsERA_obsOnly_WeatherRegime.py

```diff
@@ -1,186 +1,193 @@
 #
-# To perform a weather regime analysis using 500 mb height data.  There are 2 pre-
-# processing steps, RegridDataPlane and PcpCombine, and 4 steps in the weather regime 
-# analysis, elbow, EOFs, K means, and the Time frequency.  The elbow and K means steps 
-# begin with K means clustering.  Elbow then computes the sum of squared distances for 
-# clusters 1 - 14 and draws a straight line from the sum of squared distance for the 
-# clusters.  This helps determine the optimal cluster number by examining the largest 
-# difference between the curve and the straight line.  The EOFs step is optional.  It 
-# computes an empirical orthogonal function analysis.  The K means step uses clustering 
-# to compute the frequency of occurrence and anomalies for each cluster to give the most 
-# common weather regimes.  Then, the time frequency computes the frequency of each weather
-# regime over a user specified time frame.  Finally, stat_analysis can be run to compute
-# an categorical analysis of the weather regime classification or an anomaly correlation of
-# the time frequency data.
+# Weather regimes are defined as atmospheric patterns with high likelihoods of recurrence.
+# They can modulate many atmospheric phenomena including tornado and severe weather 
+# occurrence.  This use case computes the top 6 most frequent weather regimes using K-means 
+# clustering for the ERA observations.  It also computes the frequency of occurrence of each
+# weather regime over a 7 day period.  The code for computing weather regimes comes from Douglas Miller.
 #
+# * Miller, D. E., Wang, Z., Trapp, R. J., &Harnos, D. S., 2020: Hybrid prediction of weekly tornado activity out to Week 3: Utilizing weather regimes. Geophysical Research Letters, 47, https://doi.org/10.1029/2020GL087253. 
 
 ##############################################################################
-# Datasets
-# --------
+# Version Added
+# -------------
 #
-#  * Forecast dataset: None.
-#  * Observation dataset: ERA Reanlaysis 500 mb height.
+# METplus version 4.0.0 
 
 ##############################################################################
-# External Dependencies
-# ---------------------
-#
-# You will need to use a version of Python 3.6+ that has the following packages installed::
-#
-# * numpy
-# * netCDF4
-# * datetime
-# * pylab
-# * scipy
-# * sklearn
-# * eofs
+# Datasets
+# --------
 #
-# If the version of Python used to compile MET did not have these libraries at the time of compilation, you will need to add these packages or create a new Python environment with these packages.
+# **Forecast dataset:** None. 
 #
-# If this is the case, you will need to set the MET_PYTHON_EXE environment variable to the path of the version of Python you want to use. If you want this version of Python to only apply to this use case, set it in the [user_env_vars] section of a METplus configuration file.:
+# **Observation dataset:** ERA Reanlaysis 500 mb height for DJF 1979 - 2017.
```

Comment:
```suggestion
# **Observation:** ERA Reanlaysis 500 mb height for DJF 1979 - 2017
```
Matching the template. "datasets" is redundant since it's in the Datasets section.